README.md

```diff
@@ -1,4 +1,4 @@
-# generator-moxie-wp [![Build Status](https://travis-ci.org/moxienyc/generator-moxie-wp.svg?branch=master)](https://travis-ci.org/moxienyc/generator-moxie-wp)
+# generator-moxie-wp [![Build Status](https://secure.travis-ci.org/moxienyc/generator-moxie-wp.png?branch=master)](https://travis-ci.org/jeffreynolte/generator-moxie-wp)
 
```

Comment:
I think we can use `moxiewp` account for this one. I've enabled Travis for this repo.

Ref : https://travis-ci.org/moxienyc/generator-moxie-wp